examples/bridge-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -55,12 +55,12 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
     mCB                              = cb;
     RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
 
+    err = Platform::MemoryInit();
+    SuccessOrExit(err);
```

Comment:
Use `ReturnErrorOnFailure(Platform::MemoryInit())` here and for the below call as well. Prefer to not use `goto exit` since it forces code readers to scroll down to see the goto exit side as well for sideffects instead of relying on RAII and clear exit path.